documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -263,12 +184,16 @@ It is easy to add new domains and services to your network requester - simply fi
 How to go about this? Have a look in your nym-network-requester config directory:
 
 ```
+# network requester binary
 ls -lt $HOME/.nym/service-providers/network-requester/*/data | grep "list"
 
+# gateway with network requester binary
+ls -lt $HOME/.nym/gateways/*/data/network-requester-data | grep "list"
+
 # returns: allowed.list  unknown.list
 ```
 
-We already know that `allowed.list` is what lets requests go through. All unknown requests are logged to `unknown.list`. If you want to try using a new client type, just start the new application, point it at your local [socks client](https://nymtech.net/docs/clients/socks5-client.html) (configured to use your remote `nym-network-requester`), and keep copying URLs from `unknown.list` into `allowed.list` (it may take multiple tries until you get all of them, depending on the complexity of the application). Make sure to restart your network requester!
+We already know that `allowed.list` is what lets requests go through. All unknown requests are logged to `unknown.list`. If you want to try using a new client type, just start the new application, point it at your local [socks client](https://nymtech.net/docs/clients/socks5-client.html) (configured to use your remote `nym-network-requester`), and keep copying URLs from `unknown.list` into `allowed.list` (it may take multiple tries until you get all of them, depending on the complexity of the application). Make sure to delete the copied ones in 'unknown.list' and restart your network requester or gateway with network requester!
```

Comment:
> restart your network requester or gateway with network requester! 

Sort of linked to a comment below: I think we need to have a single term for referring to a GW+NR combo ('Smooshed Gateway'? Post-smoosh gateway?) as this reads a little confusing.

Reply:
'Exit node'? 'Exit gateway'?

Reply:
I like 'Exit Gateway' actually

Reply:
Also you refer to it as 'Exit gateway' above - lets keep this phrasing and have it throughout. We can add that the 'exit gateway' is a gw + nr either in a note at the top of the page, or just have it in brackets after the definition for the moment so people get used to it.

Reply:
i go with exit gateway everywhere to entrench it.